<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>&lt;aumuell@reserv.at&gt;</author>
 <class>OrganizeCollectionDialogBase</class>
 <widget class="QWidget" name="OrganizeCollectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Organize Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="folderLabel">
       <property name="toolTip">
        <string>Base directory under which to put files</string>
       </property>
       <property name="text">
        <string>C&amp;ollection Folder:</string>
       </property>
       <property name="buddy">
        <cstring>folderCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="folderCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The collection folder to which the files will be transferred.</string>
       </property>
       <property name="whatsThis">
        <string>The collection folder to which the files will be transferred.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="optionsButton">
       <property name="toolTip">
        <string>&lt;html&gt;Show/hide additional controls to specify destination track paths based on their metadata.&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;Show/hide additional controls to specify destination track paths based on their metadata.&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Options</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-down"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="OrganizeCollectionWidget" name="organizeCollectionWidget" native="true"/>
   </item>
   <item>
    <widget class="OrganizeCollectionOptionWidget" name="optionsWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="previewBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Destination Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="previewTableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Preview: &lt;placeholder&gt;</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeft|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Original: &lt;placeholder&gt;</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeft|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="conflictLabel"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="overwriteCheck">
          <property name="toolTip">
           <string>If checked, overwrite files of the same name without asking.</string>
          </property>
          <property name="whatsThis">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Check this to overwrite files of the same name automatically. &lt;span style=&quot; font-style:italic;&quot;&gt;(Warning: Overwriting is not undoable)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Overwrite &amp;Destination</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>OrganizeCollectionWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/OrganizeCollectionDialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrganizeCollectionOptionWidget</class>
   <extends>QGroupBox</extends>
   <header>dialogs/OrganizeCollectionDialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">qlineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
